### PR TITLE
WFCORE-1194 Name for wildcard resources should be configurable in parser

### PR DESCRIPTION
**Pull request by ctomc**

### CODE REVIEW

File: controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescription.java [hunk 1]
```diff
@@ -518,6 +522,11 @@ public PersistentResourceXMLBuilder setMarshallDefaultValues(boolean marshallDef
             return this;
         }
 
+        public PersistentResourceXMLBuilder setNameAttributeName(String nameAttributeName) {
```
Comment (bstansberry) [inline controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescription.java]: This one could use some javadoc as it's not so obvious what it's about. :)

**Comment by bstansberry** [comment]
Javadoc bit aside, looks fine.

**Comment by ctomc** [comment]
javadoc was added.

**Comment by darranl** [comment]
:warning: It has been a while since we talked about this one so just adding a note to say this is the one we need to be sure the SHA of the commit does not change as it is included in a shared topic branch.

**Comment by ctomc** [comment]
I do not plan to change SHA of this commit. 
This should be merged quite soon, given it got all acks for inclusion in EAP.

**Pull request merged**